packages/core/core/src/ValidatorRunner.js

```diff
@@ -0,0 +1,176 @@
+// @flow
+
+import type {
+  Blob,
+  File,
+  FilePath,
+  Validator,
+  AssetRequest,
+  ParcelOptions
+} from '@parcel/types';
+
+import path from 'path';
+import {
+  md5FromFilePath,
+  md5FromReadableStream,
+  md5FromString
+} from '@parcel/utils';
+import Cache from '@parcel/cache';
+import {TapStream} from '@parcel/utils';
+import {createReadStream} from 'fs';
+
+import Dependency from './Dependency';
+import Config from './Config';
+import ResolverRunner from './ResolverRunner';
+import {report} from './ReporterRunner';
+import {MutableAsset} from './public/Asset';
+import InternalAsset from './Asset';
+
+type Opts = {|
+  config: Config,
+  options: ParcelOptions
+|};
+
+const BUFFER_LIMIT = 5000000; // 5mb
+
+export default class validatorRunner {
```

Comment:
hmm a bunch of stuff in here seems duplicated from transformers... wonder what we can reuse.

Reply:
Most of it's copy paste from transformerrunner